server/utils/statusEnum.js

```diff
@@ -8,9 +8,10 @@ const states = {
   APPROVED: { name: 'APPROVED', value: 'Approved' },
 }
 states.SECURITY_MANUAL.previous = [states.STARTED, states.SECURITY_AUTO, states.SECURITY_BACK]
-states.SECURITY_AUTO.previous = [states.STARTED]
+states.SECURITY_AUTO.previous = [undefined, states.STARTED]
```

Comment:
can remove started from the previous section when auto trigger is moved in later ticket